3_javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,68 @@
+
+const ApiKey = `https://open.er-api.com/v6/latest/EUR`;
```

Comment:
Since you are not using any variable inside the string, you can just use normal single quotes (').